### DISCUSSION TITLE
Fork build failing: Missing 'lsd:module IRI' in multiple package.json files

### DISCUSSION THREAD

**Discussion by zg009** [comunica/comunica#1094] [Q&A]
Hello, I tried to fork the comunica project and build it using yarn v3.2.4 (with node v16.13.1) and received multiple Missing lsd:module IRI errors which I believe caused the build to fail? I understand the componentsjs framework somewhat but I'm unsure how to fix this.

 I tried looking for other discussions and didn't find any, though please point me to one if I missed it.
![Capture](https://user-images.githubusercontent.com/103973669/194464015-fd15c9ec-d19b-4ece-9096-bb0551782e67.JPG)

**Comment by rubensworks** [comment]
Those are just warnings, so you can ignore them.
However, those packages should actually be ignored from component generation: https://github.com/comunica/comunica/blob/master/.componentsjs-generator-config.json#L3-L16
Might be something Windows-specific that causes this to break.

Could you report a bug for this on the Components.js issue tracker?

**Reply by jeswr** [reply]
>  Might be something Windows-specific that causes this to break.

The change in slash direction with `\comunica/` certainly jumps out at me
